src/olympia/pages/urls.py

```diff
@@ -47,4 +47,10 @@
         TemplateView.as_view(template_name='pages/sunbird.html'),
         name='pages.sunbird'),
     url('^sunbird/', lambda r: perma_redirect(reverse('pages.sunbird'))),
+
+    url('^shield-study-1$',
+        TemplateView.as_view(template_name='pages/shield_study_1.html'),
+        name='pages.shield_study_1'),
+    url('^shield-study-1/',
+        lambda r: perma_redirect(reverse('pages.sunbird'))),
```

Comment:
Copy/paste fail :)

I'm not sure you need that. Couldn't you just include a trailing slash in the true URL instead ?